src/app/public/modules/data-manager/data-manager.service.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+import {
+  SkyDataManagerConfig,
+  SkyDataManagerState,
+  SkyDataManagerStateChange,
+  SkyDataViewConfig,
+  SkyDataViewState
+} from './models';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import {
+  filter,
+  map
+} from 'rxjs/operators';
+
+@Injectable()
+export class SkyDataManagerService {
+
+  private readonly activeViewId: BehaviorSubject<string> = new BehaviorSubject<string>(undefined);
```

Comment:
Should we be completing these `BehaviorSubjects` when the service gets destroyed?